src/HeaderSort/HeaderSort.d.ts

```diff
@@ -0,0 +1,13 @@
+import { ListView as SharedListView } from "../Shared/SharedUtils";
+
+export interface HeaderSortListView extends SharedListView {
+    friendlyId: string;
+}
+
+interface WrapperProps {
```

Comment:
Add this interface to the `Shared/SharedUtils` since its used in all `xxxxxx-container` components.